src/UI/Component/ViewControl/Factory.php

```diff
@@ -64,4 +64,38 @@ public function mode($labelled_actions, $aria_label);
 	 */
 	public function section(Button $previous_action, Button $button, Button $next_action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The sortation view control enables users to change the order in which
+	 *      some data is presented.
+	 *      The common table-sorting mechanism, e.g., in which a column header is clicked,
+	 *      requires the column to be visible. Visibility and sorting can be decoupled
```

Comment:
And can only be used if you have table like structures at all! This is not always the case (e.g. Appointment Listings or similar).